### PR TITLE
Removed empty space

### PR DESCRIPTION
**Pull request by edworczak**
## Description
Removed empty space on popover when there is no comment.
Provided different logic and icons for comment and when there is no comment.
Introduced Font Awesome.
Improved buttons UX in editing comments.

---
I hereby agree to the terms of the AET Contributor License Agreement.

### CODE REVIEW

**Comment by michaldziubek** [comment]
It looks really good!
Just one small thing - please update the CHANGELOG.md file with your PR description.

File: documentation/src/main/wiki/LinuxAndWindowsSetup.md [hunk 1]
```diff
@@ -8,81 +8,75 @@ Please note that full list of required tools and its versions can be found in [S
...
-10. Unpack nssm-2.24 under *C:\aet*.
-11. Install Karaf as Windows service:
-    * run `C:\aet\nssm-2.24\win64\nssm.exe install apache-karaf`
-    * NSSM prompt should be displayed. Add “C:\aet\karaf\current\bin\karaf.bat” as Application Path. Startup dir should be filled up automatically to “C:\aet\karaf\current\bin”
-12. Unpack “browsermob-proxy-2.0.0-bin” to PATH *C:\aet*.
-13. Install Browsermob as Windwos service:
-    * run `C:\nssm-2.24\win64\nssm.exe install browserMob-proxy`
-    * NSSM prompt should be displayed. Add “C:\aet\browsermob-proxy-2.0.0\bin\browsermob-proxy.bat” as Application Path. Startup dir should be filled up automatically to “C:\aet\browsermob-proxy-2.0.0\bin”
-15. Install Firefox Setup 38.6.0esr:
+5. Unpack [Apache Karaf 4.1.4](https://archive.apache.org/dist/karaf/4.1.4/apache-karaf-4.1.4.zip) to C:\aet\karaf.
```
Comment (mateuszluczak) [inline documentation/src/main/wiki/LinuxAndWindowsSetup.md]: Are this changes expected to be part of this branch?

File: core/worker/src/main/java/com/cognifide/aet/worker/drivers/WebDriverConstants.java [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.cognifide.aet.worker.drivers;
+
+/**
+ * Some constants used by web driver factory implementations.
+ */
+public final class WebDriverConstants {
```
Comment (mateuszluczak) [inline core/worker/src/main/java/com/cognifide/aet/worker/drivers/WebDriverConstants.java]: Are this changes expected to be part of this branch?

**Comment by wiiitek** [comment]
Closing this pull request as the branch was corrupted with SourceTree.

New branch with changes is [note-tooltip-new](https://github.com/Cognifide/aet/tree/note-tooltip-new)

However it still requires some changes...

**Pull request closed without merging**